charts/karavi-observability/values.yaml

```diff
@@ -119,7 +119,7 @@ karaviMetricsPowerscale:
     # sidecarProxy.image: the container image used for the csm-authorization-sidecar.
     # Default value: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
```

Comment:
Can you update the version in this comment as well

Reply:
Done